packages/popover/README.md

```diff
@@ -59,17 +59,17 @@ The popover component will be automatically positioned relative to its nearest p
+| `align`            | `'top'` \| `'bottom'` \| `'left'` \| `'right'` \| `'center-horizontal'` \| `'center-vertical'` | A string that determines the alignment of the popover relative to the `refEl`.                                                                                                                                           | `'bottom'` |
+| `justify`          | `'start'` \| `'middle'` \| `'end'` \| `'fit'`                                                  | A string that determines the justification of the popover relative to the `refEl`. Justification will be defined relative to the `align` prop                                                                            | `'start'`  |
+| `children`         | `node`                                                                                         | Content that will appear inside of the `<Popver />` component                                                                                                                                                            |            |
+| `usePortal`        | `boolean`                                                                                      | Will position Popover's children relative to its parent without using a Portal, if `usePortal` is set to false. NOTE: The parent element should be CSS position `relative`, `fixed`, or `absolute` if using this option. | `true`     |
+| `spacing`          | `number`                                                                                       | Specifies the amount of spacing (in pixels) between the trigger element and the content element.                                                                                                                         | `10`       |
+| `className`        | `string`                                                                                       | Classname to apply to popover-content container                                                                                                                                                                          |            |
+| `adjustOnMutation` | `boolean`                                                                                      | Should the Popover auto adjust its content when the DOM changes (using MutationObserver).                                                                                                                                | `false`    |
+| `onClick`          | `function`                                                                                     | Function that will be called when popover content is clicked.                                                                                                                                                            |            |
 
 ## Advanced Use Case
```

Comment:
github won't let me comment on the specific line, but a few lines below we define the type of `refEl` to be `object`

Reply:
Updated to HTMLElement